plugins/config_manager.py

```diff
@@ -223,5 +223,6 @@ def sanitize_url_to_fs(url):
         csum = dnf.yum.misc.Checksums(['sha256'])
         csum.update(url[lastindex:])
         url = url[:lastindex] + '_' + csum.hexdigest()
-
-    return url
+    # remove all not allowed characters
+    allowed_regex = "[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.:]"
```

Comment:
dash '-' is misplaced here. It must be either escaped or placed as first/last character:
`"[^\w.:-]"` is shorter and equivalent